### ISSUE TITLE
No implementation found for method startHeadlessService 

### ISSUE THREAD

**Issue by KieranLafferty** [luanpotter/audioplayers#306]
I'm sure this is a known issue for anyone using latest master but on Android I get the following error on startup: 

```
"No implementation found for method startHeadlessService on channel xyz.luan/audioplayers"
```

I was using the latest master since our product requires lock screen controls which aren't currently available in any of the public releases. 

Totally understand that I'm using bleeding edge and that these kind of errors are to be expected just thought I'd bring attention to it in case it hadn't been seen yet

**Comment by dvorapa** [comment]
This should be a warning, that setting playback rate does not work on Android yet, but the warning message seems to be not really good

**Comment by eligt** [comment]
I'm experiencing this exact issue on 0.13.3. I can see that this version was released only 5 days ago and the commit above is from 3 days ago, any plans on releasing 0.13.4 relatively soon? I'd like to avoid using master just in case any unstable changes are introduced.

**Comment by zayngdev** [comment]
Got the same issue

**Comment by joshhaaronn** [comment]
This issue doesn’t seem to be specific to Android. I’m having the same issue with 13.3 on iOS

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by eligt** [comment]
I realized that I failed to mention in my prior comment that this can be resolved by rolling back to a previous version of the plugin. Just replace the dependency in your `pubspec.yaml` with the following:
```yaml
dependencies:
  audioplayers: 0.13.2
```

**Comment by erickzanardo** [comment]
Fixed on #332
